### PR TITLE
new layout

### PR DESCRIPTION
**Pull request by ronnaf**
Updates:
- update chakra theme and fonts (from fontsource)
- update chakra version and its other deps
- add typescript support
- update changelog hub layout
- update individual changelog: just wrap it with container, replace navbar and footer
- replace moment with lighter dayjs

Screenshots:

Event digest - June Changelog 2-12-2022 9-00-47 PM
Changelog - June Changelog 2-12-2022 9-00-33 PM
New users - June Changelog 2-12-2022 9-03-01 PM
Changelog - June Changelog 2-12-2022 9-02-34 PM

### CODE REVIEW

File: frontend/components/core/footer/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import { FooterLink } from "./footer-link";
+import { FooterTitle } from "./footer-title";
+
+const gap = [2, 2, 8];
+
+interface Props {
+  _wrapper?: ContainerProps;
+}
+
+export function FooterV2(props: Props) {
```
Comment (ronnaf) [inline frontend/components/core/footer/footer.tsx:21]: we're still missing a few links for the footer

File: frontend/components/core/navbar/navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+import React, { useState } from "react";
+import { defaultPx } from "../../../lib/utils/default-container-px";
+import { CloseIcon } from "../custom-icons/close-icon";
+import { HamburgerMenu } from "../custom-icons/hamburger-icon";
+import { DesktopNavItem } from "./desktop-nav-item";
+
+const MOBILE_MENU_COLOR = "#241f47";
+const MOBILE_FONT_WEIGHT = 600;
+
+const ROUTES = [
```
Comment (ronnaf) [inline frontend/components/core/navbar/navbar.tsx:21]: need to check if /solution and /vision is correct

**Comment by ferrucc-io** [comment]
looking good, waiting for all new links to exist before merging

**Comment by ronnaf** [comment]
@ferrucc-io navbar links and footer links summary:

# june changelog links

## navbar links

- Solution - process.env.NEXT_PUBLIC_MARKETING_HOST + "/solution" - leads to nothing
- Templates - process.env.NEXT_PUBLIC_MARKETING_HOST + "/templates"
- Pricing - process.env.NEXT_PUBLIC_MARKETING_HOST + "/pricing"
- Vision - process.env.NEXT_PUBLIC_MARKETING_HOST + "/vision" - leads to nothing

## footer links

Solution

- Overview - [https://june.so/](https://june.so/)
- Templates - [https://june.so/](https://june.so/)templates
- Pricing - [https://june.so/](https://june.so/)pricing
- Changelog - /
- Widget for iOS - ???

For

- Founders - [https://june.so/](https://june.so/)personas/founders
- Product Managers - [https://june.so/](https://june.so/)personas/product-managers
- Engineers - [https://june.so/](https://june.so/)personas/engineers
- Marketers - [https://june.so/](https://june.so/)personas/marketers
- Customer Success - [https://june.so/](https://june.so/)personas/customer-success
- Designers - [https://june.so/](https://june.so/)personas/designers

Company

- Our story - ???
- Careers - [https://www.notion.so/projectanalytics/Work-at-June-ba2ff41d03cb4a1ba230eda21daccada](https://www.notion.so/Work-at-June-ba2ff41d03cb4a1ba230eda21daccada)
- Contact - [mailto:enzo@june.so](mailto:enzo@june.so)
- Twitter - [https://twitter.com/JuneDotSo](https://twitter.com/JuneDotSo)

Legal

- Terms - [https://www.notion.so/Terms-of-Service-3a8be1c25dd04cf699800425153a03f8](https://www.notion.so/Terms-of-Service-3a8be1c25dd04cf699800425153a03f8)
- Privacy - [https://www.notion.so/Privacy-Policy-a4f99393a98b4ce6aa1bacd5f48157cc](https://www.notion.so/Privacy-Policy-a4f99393a98b4ce6aa1bacd5f48157cc)

**Comment by ferrucc-io** [comment]
iOS widgets should lead to: https://widgets.june.so/

**Pull request merged**